Week1/Section2/Exercise7/Source7.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+Exercise 7 - Rock, Paper, Scissors
+Write a simple rock, paper, scissors game.
+Ask the user for their guess on the result, then tell them whether they guessed right or wrong.
+
+Example output
+Welcome to Rock, Paper, Scissors!
+
+1 - Rock
+2 - Paper
+3 - Scissors
+Choose your weapon: 2
+
+Your choice: Paper
+Computer�s choice: Rock
+
+You win� big deal!
+*/
+
+#include <iostream>
+
+enum options {
```

Comment:
An enum name is clearer if you don't make it plural. So `options` could work better than `option`. This is because you can declare a variable of this type, so it would be weird to have:
```
options userChoice = rock;
```
Since your variable holds just one value.